### PR TITLE
Product page

### PR DESCRIPTION
**Pull request by prathameshpcodes**

### CODE REVIEW

**Review by vivvvek — COMMENTED** [review]
there is an issue with padding of the footer & product as well

File: pages/product.html [hunk 1]
```diff
@@ -0,0 +1,469 @@
...
+
+  <body>
+    <!-- header starts -->
+    <header>
+      <!-- name & logo -->
+      <nav class="container flex justify-c-sb alg-i-ctr px-2 py-1">
+        <div class="navbar-align">
+          <!-- logo -->
+
+          <!-- <i class="fa-solid fa-bat text-red"></i> -->
```
Comment (varun1233) [inline pages/product.html]: Remove commented code if not required

**Comment by kaustubh-jsr** [comment]
I can just see the index page, if there is a separate product page, it is not visible.
Rest the UI and Code looks good to me.

**Comment by prathameshpcodes** [comment]
> I can just see the index page, if there is a separate product page, it is not visible. Rest the UI and Code looks good to me.

https://deploy-preview-2--musing-boyd-67a39d.netlify.app/pages/product.html , check this out

**Review by varun1233 — APPROVED** [review]
Suggestion: You can have a bit more descriptive css naming and remove commented code wherever not required. 
Apart from that code looks good

File: css/pages/product.css [hunk 2]
```diff
@@ -0,0 +1,28 @@
+.filter-container {
+  background: var(--color-white);
+
+  box-shadow: 0 0.1rem 0.4rem var(--color-slate-200);
```
Comment (kaustubh-jsr) [inline css/pages/product.css:4]: You can use px instead of rems for units less than 10px. This isn't mandatory, just a suggestion.

**Review by kaustubh-jsr — COMMENTED** [review]
Code is nicely formatted. UI looks good to me. You can merge.

File: css/pages/product.css (hunk 2 above)
Comment (prathameshpcodes) [reply css/pages/product.css:4]: Thanks for the suggestion,

**Pull request merged**